aws/ecs/packer/to-upload/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -eu
+
+LOGGER_TAG=""
+
+is_container_running() {
+    CONTAINER_NAME=$1
+    [ "$(docker inspect -f '{{.State.Running}}' $1 2> /dev/null)" = true ]
+}
+
+succeed_or_die() {
+    if ! OUT=$($@ 2>&1); then
+	echo "error: $1 failed: $OUT" | logger -p local0.error -t $LOGGER_TAG
+	exit 1
+    fi
+}
+
+run_scope() {
+    while true; do
+	# verify that scope is not running
+	while is_container_running weavescope; do sleep 2; done
+	# launch scope
+	ARGS=""
+	if [ -e /etc/weave/scope.config ]; then
```

Comment:
Uhm, yep, you are probably right. Where we put the stuff calls for a different pull request though (since it also affects the ECS guide)